CHANGELOG.rst

```diff
@@ -6,6 +6,8 @@ in development
 
 Added
 ~~~~~
+* EL6 deprecation. Removed building of EL6 RPMs and associated jobs
+  Contributed by Amanda McGuinness (@amanda11 Ammeon Solutions)
```

Comment:
This fits into `Changed` sub-category under the Changelog. Please take a look at examples from the previous release. Also please try to format this changelog message based on those examples.

Reply:
Sorry - didn't see the different sub-sections.
I see in 3.2.0 there actually was a Removed sub-category with "Removed Ubuntu 14.04 from test matrix #4897" in it.
Would you prefer it in Changed or Removed?

Reply:
Yes absolutely, `Removed` is even better :+1:
Didn't know we had that section. Thanks for noticing that!